handlers/bot_handler.py

```diff
@@ -54,14 +54,36 @@ def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_ad
             raise InvalidCommand("Sorry, couldn't retrieve the git information for the bot...")
 
 
+class InviteCommand(Command):
+    """
+    Invite a list of members to the current channel, parses out members already
+    present
+    """
+
+    @classmethod
+    def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_admin):
+        """Execute the Invite command"""
+        try:
+            current_members = slack_wrapper.get_channel_members(channel_id)
+            # strip uid formatting
+            invited_users = [user.strip("<>@") for user in args]
+            # remove already present members
+            invited_users = [user for user in invited_users if user not in current_members]
```

Comment:
is this necessary? what happens if we just invite everyone? I imagine the api will ignore the ones that are already in the channel?

Reply:
api will reply with errors for users present

this approach saves a few api calls, which is better for big lists in case of rate limiting